Web/Pages/CCGMember/AllPositions.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@page 
+@inject IViewLocalizer Localizer
+@model Web.Pages.CCGMember.AllPositionsModel
+@{
+    var isEnglish = System.Threading.Thread.CurrentThread.CurrentCulture.TwoLetterISOLanguageName == "en";
+    var jobGroups = Model.JobGroups;
+    var imagefolder = "~/img/CCGMember/";
+    bool noresults = true;
+}
+<section role="contentinfo">
+    <div class="col-md-12">
+        <h2><b>@Localizer["AllPositions"]</b></h2>
+        <form>
+            <p>
+                <input type="text" asp-for="Filter" />
+                <input type="submit" text=@Localizer["Keywords"] value=@Localizer["Filter"] />
+            </p>
+        </form>
+        <form method="post">
+
+        @foreach (var jobgroup in jobGroups.Where(e => Model.Filter == null || e.NameEng.ToLower().Contains(@Model.Filter.ToLower())))
```

Comment:
You could move the filtered list to a variable. Then it could be used in foreach loop and replace noresult with .Any().